hazelcast/src/main/java/com/hazelcast/internal/server/tcp/TcpServerConnectionManager.java

```diff
@@ -128,13 +128,15 @@ public ServerConnection get(@Nonnull Address address, int streamId) {
     @Override
     @Nonnull
     public List<ServerConnection> getAllConnections(@Nonnull Address address) {
-        UUID uuid = addressRegistry.uuidOf(address);
+        UUID instanceUuid = addressRegistry.uuidOf(address);
         // Because duplicate connections can be established on the planes and
         // we don't keep all duplicates on the planes, we need to iterate over
         // connections set which stores all active connections.
-        return uuid != null
-                ? connections.stream().filter(connection -> connection.getRemoteUuid().equals(uuid))
-                                      .collect(Collectors.toList())
+        return instanceUuid != null
+                ? connections.stream().filter(connection -> {
+                    UUID remoteUuid = connection.getRemoteUuid();
```

Comment:
Maybe it will be more concise to use Objects.equals()?
If _instanceUuid_ is not _null_, 
_Objects.equals(instanceUuid, connection.getRemoteUuid())_
could replace those 2 lines.